common/interface.go

```diff
@@ -289,6 +289,11 @@ type EnableEpochsHandler interface {
 	RefactorPeersMiniBlocksEnableEpoch() uint32
 	GetCurrentEpoch() uint32
 
+	IsFlagDefined(flag core.EnableEpochFlag) bool
+	IsFlagEnabledInCurrentEpoch(flag core.EnableEpochFlag) bool
```

Comment:
can we also rename the IsFlagEnabledInCurrentEpoch into IsFlagEnabled?

Reply:
same, will be done on the next PR